pkgs/top-level/perl-packages.nix

```diff
@@ -21202,6 +21202,20 @@ let
     };
   };
 
+  XMLRPCLite = buildPerlPackage {
+    pname = "XMLRPC-Lite";
+    version = "0.717";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/P/PH/PHRED/XMLRPC-Lite-0.717.tar.gz";
+      sha256 = "0925md6jhzgpsibwgny4my461b2wngm8dhxlcry8pbqzrgrab7rs";
+    };
```

Comment:
```suggestion
    };
    # Remove network test which is not skipped correctly on darwin
    preCheck = "rm t/26-xmlrpc.t"; 
```

Reply:
Should be resolved now, gotta wait for build though.